for_testing.py

```diff
@@ -0,0 +1,130 @@
+import sys

```

Comment:
`point_layer = QgsProcessingUtils.mapLayerFromString ("/D:/M.Tech/Fifth `Sem/MTP/snr/projection.shp",context)`

I guess this line of code will provide me with the bus termini, but this doesn't seem right because I can get termini from another shapefile that contains only bus termini.

`network_layer = QgsProcessingUtils.mapLayerFromString ("/D:/M.Tech/Fifth Sem/MTP/snr/Transport_Road.shp",context)`
  This is for creating a network because it contains the Nashik district road network.
  
  And I have another data that is called unique termini set. 
  
  My aim is to create a Route network by following the way.
  
  Iterate over Unique Route segment to input **source and destination**
  Now get that source and destination Geom from Termini
  
  Then create line by using **Dijkstra's algorithm** and it should use the **Road Network**
  
  
  To run the code, I just copied it and pasted it to one qgis python console and ran it.